lesson-16.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/assets/styles/stylesl-16.css">
+</head>
+<body>
+    <div class="grid-container">
+        <div class="div-header">
+            <a href="#">Home</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+         </div>
+         <div class="div-leftbar">
+            <a href="#">Product</a>
+            <a href="#">Services</a>
+            <a href="#">Users</a>
+            <a href="#">Feedback</a>
+        </div>
+        <div class="div-rightbar">
+            <img src="/assets/img/dog9.webp" alt="Кот 1">
+            <img src="/assets/img/cat10.webp" alt="Кот 2">
+            <img src="/assets/img/dog11.webp" alt="Кот 3">
+            <div class="image-container2">           
+            <img src="/assets/img/dog12.webp" alt="Кот 4">
+            <div class="overlay2">Woof!</div>
+            </div>
+        </div>
+        <div class="div-main">
+            <img src="/assets/img/cat1.webp" alt="Кот 1" class="large-image">
```

Comment:
In order to make the images responsive and inside the grid, better wrap each of them into a container. And position accordingly the container. And after that just set "object-fit: cover" to the image inside container.

Reply:
wrap the all in image-container as you did with the last one